application/web/index.html

```diff
@@ -104,7 +105,7 @@ <h2 id="journal-title"></h2>
                                 <span>Save Entry</span>
                             </button>
                             <button id="three-dots-entry" class="three-dots">
-                                <img src="./assets/vdots-grey.svg" alt="Three Dots" class="dots-icon">
+                                <img src="./assets/vdots-grey.svg" alt="Three Dots" >
```

Comment:
Just some standard stuff but could you remove the whitespace after the "Three Dots"